specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2022-04-13-preview/vi.json

```diff
@@ -0,0 +1,1280 @@
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Azure Video Indexer account was successfully deleted."
+          },
+          "202": {
+            "description": "The Azure Video Indexer account will be deleted asynchronously."
+          },
+          "204": {
+            "description": "The Azure Video Indexer account does not exist."
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete account": {
+            "$ref": "./examples/ViAccountDelete.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.VideoIndexer/locations/{location}/userClassicAccounts": {
+      "get": {
+        "tags": [
+          "Accounts"
+        ],
+        "operationId": "UserClassicAccounts_List",
+        "description": "Lists all Azure Video Indexer classic accounts",
+        "x-ms-examples": {
+          "List accounts": {
+            "$ref": "./examples/ViListUserClassicAccounts.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/UserClassicAccountList"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.VideoIndexer/locations/{location}/classicAccounts/{accountName}": {
+      "get": {
+        "tags": [
+          "Accounts"
+        ],
+        "operationId": "ClassicAccounts_GetDetails",
```

Comment:
Why do we not name this operation `ClassicAccounts_Get` as other resources?

Reply:
We have another operation that lists classic account. We don't want this operation to be mixed up with the other. ClassicAccount_Get  could suggest that we want to get all classic account. where this name is specific to get the details of a classic account